apps/condo/domains/billing/hooks/useReceiptTableColumns.ts

```diff
@@ -5,13 +5,17 @@ import { getFilterIcon, getTextFilterDropdown } from '@condo/domains/common/comp
 import { getMoneyRender, getTextRender } from '@condo/domains/common/components/Table/Renders'
 import { useIntl } from '@open-condo/next/intl'
 import get from 'lodash/get'
+import { FiltersMeta, getFilterDropdownByKey } from '../../common/utils/filters.utils'
```

Comment:
if the import is from the other domain, then we use relative import
```suggestion
import { FiltersMeta, getFilterDropdownByKey } from '@condo/domains/common/utils/filters.utils'
```